Makefile

```diff
@@ -80,10 +80,10 @@ release-install: release-container
 
 verify-shell: docker-running
 	docker run -ti --rm \
-		--privileged \
-		--volume /home/cockpit:/home/user \
+		--privileged --uts=host \
+		--volume /opt/cockpit-verify-secrets:/secrets \
```

Comment:
IMHO this is a bit of an abuse of [/opt](http://www.pathname.com/fhs/2.2/fhs-3.12.html). `/opt/cockpit-verify/` is somewhat cleaner, with `secrets` and `images` subdirectories, but it's still not a self-sustained third-party app there. This is what `/var` is for. But I won't dwell on this.

Also, files in /opt would normally be root rowned, but the container chowns them to user '1111' which doesn't exist (any more) on the host. This can easily be avoided for the secrets (they can be copied), but not for the images (they are too big and need to be written).

Can the secrets dir be mounted read-only, to ensure that our container doesn't mess around with them? We would never want to ruin a real user's ssh key. The images are less critical as they are public and can always be restored easily (i. e. `/images` is just a cache).